runtime/oti/jvminit.h

```diff
@@ -453,6 +453,9 @@ enum INIT_STAGE {
 #define VMOPT_XXTRANSPARENT_HUGEPAGE "-XX:+TransparentHugePage"
 #define VMOPT_XXNOTRANSPARENT_HUGEPAGE "-XX:-TransparentHugePage"
 
+#define VMOPT_XXDEEP_SCAN "-XX:+DeepStructurePriorityScanGC"
```

Comment:
Note a strong opinion on this one but I the option reads better if the GC is put at the start rather than the end.
`-XX:+-GCDeepStructurePriorityScan`